pyproject.toml

```diff
@@ -39,7 +39,7 @@ server = [
   "zeroconf==0.131.0",
 ]
 test = [
-  "black==23.12.1",
+  "black==24.2.0",
```

Comment:
I think we should remove black from requirements. We use ruff for formatting now.

Reply:
https://github.com/home-assistant-libs/python-matter-server/pull/555